pkg/ksym/ksym_test.go

```diff
@@ -168,11 +168,6 @@ ffffffff8f6d15c4 a not_in_order
 		addr2 + 1: "xfrm_km_lock",
 	}, syms)
 
-	require.Equal(t, map[uint64]string{
-		addr1 + 1: "tcp_sock_id",
-		addr2 + 1: "xfrm_km_lock",
-	}, c.fastCache)
```

Comment:
Removing it and the below, as this is testing an implementation detail. We already make sure that we hit the cache in a test for this purpose